src/gurt/tests/test_gurt.c

```diff
@@ -1028,6 +1038,7 @@ static d_hash_table_ops_t th_ops_ref = {
 	.hop_rec_decref		= test_gurt_hash_op_rec_decref,
 	.hop_rec_ndecref	= test_gurt_hash_op_rec_ndecref,
 	.hop_rec_free		= test_gurt_hash_op_rec_free,
+	.hop_key_get		= test_gurt_hash_op_key_get,
```

Comment:
I removed this function a few time ago. Why you use old version? Please rebase on top of current master.

Reply:
This function is necessary for dynamic hash in order to associate to find object. In the existing hash, the object type is d_list_t* and could be removed from the hash directly. However, in the dynamic hash, the key is necessary to find the object if d_hash_rec_decref(),  d_hash_rec_ndecref(), forces object to get removed.